packages/components/CONTRIBUTING.md

```diff
@@ -22,121 +17,3 @@ $tc-{component-name}-{variable-name}
 *variable-name* the specific name
```

Comment:
Why do we need an extra CONTRIBUTING.md file here, we have same rules for _/packages/forms_ too?